examples/misc/test/library_tour/core_test.dart

```diff
@@ -212,10 +212,7 @@ void main() {
   group('Collections: List:', () {
     test('constructor', () {
       // #docregion List
-      // Use a List constructor.
-      var vegetables = List();
```

Comment:
I'm curious why we are deleting the empty list constructor. I'd argue for writing it as `<type>[]` instead tho.

Reply:
We could definitely add a new example using an empty list literal, I was just trying to match the old example with the new one. I definitely think that would be worth it and in a separate PR we can add a new empty example back.